packages/eslint-plugin-kolibri/lib/rules/vue-no-unused-methods.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview Disallow unused methods.
+ */
+
+'use strict';
+
+const remove = require('lodash/remove');
+const eslintPluginVueUtils = require('eslint-plugin-vue/lib/utils');
+
+const utils = require('../utils');
+const constants = require('../constants');
+
+const { GROUP_METHODS } = constants;
```

Comment:
Not necessary to change, but if `GROUP_METHODS` is the only thing you are using from the `constants` variable, then you can write these two lines as : `const { GROUP_METHODS } = require('../constants');` to do the destructuring and assignment in one line.

Reply:
Right, thanks, will update it tomorrow.